app/controllers/api/v1/books_controller.rb

```diff
@@ -1,8 +1,7 @@
 class Api::V1::BooksController < Api::V1::BaseController
   # GET /books
   def index
-    @books = Book.paginate(page: params[:page], per_page: 20)
-    render json: @books
+    head(:not_found)
```

Comment:
I would suggest using an include/exclude in the routes instead of returning a 404 for each controller index method. Then just move these. For eg.

`resources :books, :except => [:index]`

Reply:
I did that at first, but a rails error was returned. I thought returning a 404 was a better solution. I'm not sure what happens in production mode.

Reply:
Done